Gaijinizer patch/CommonEvents_600.txt

```diff
@@ -17,11 +17,10 @@ Beetle Harpy Queen is back in shape......
 
 \\C[2]もう後が無い……！！
 > CONTEXT: CommonEvents_600/70/Dialogue 
-\\N[1] is grabbed by Beetle Harpy,
-It will be carried to the chest of the
-queen gradually ...!
+\\N[1], shoulders held by the Harpy Spawn, is
+dragged to the Queen's chest!
 
-\\C[2] There are no more ......! !#GT
+\\C[2]There's no more time...!!#Amaboye
 
 > END STRING
```

Comment:
There's an unneeded blank line at the end, and we can revise this a bit for flow:
```
\\N[1] is dragged to the Queen's chest, his shoulders
gripped by Harpy Spawn.

\\C[2]There's no more time...!!
```